registry/tests/templates.spec.ts

```diff
@@ -273,7 +273,7 @@ describe(`Tests ${example.url}`, () => {
             }
         });
 
-        it('should filter a list of templates by domain', async () => {
+        it.only('should filter a list of templates by domainId using', async () => {
```

Comment:
please avoid only and if possible catch it in lint

Reply:
sure thing. BTW, it's already removed in the recent commits.

I will create an issue for adding eslint (we don't have it in `ilc` AFAIK).